__tests__/mainTestSuite.test.js

```diff
@@ -112,25 +112,60 @@ describe('GET /api/articles', () => {
       })
     })
   });
-  it('should respond with an array based on use with the author query', () => {
-      return request(app)
-      .get('/api/articles?author=rogersop')
-      .expect(200)
-      .then((response) => {
-        const {articles} = response.body
-        expect(articles.length).toBe(3)
-        expect(articles).toBeSortedBy('author')
-      })
-  });
-  it('should respond with an array based on use with the topic query', () => {
+  // it('should respond with an array based on use with the author query', () => {
```

Comment:
Don't leave commented out code that is doing nothing in work you want reviewing. Remove it.